Demo/Application/Base/AppDelegate.swift

```diff
@@ -0,0 +1,81 @@
+import Foundation
+
+@objcMembers class AppDelegate: UIResponder, UIApplicationDelegate {
+        
+    let BraintreeDemoAppDelegatePaymentsURLScheme = "com.braintreepayments.Demo.payments"
```

Comment:
```suggestion
    let returnURLScheme = "com.braintreepayments.Demo.payments"
```

Reply:
Addressed in dc311d2bb8860d88472b04f886b40b4358ac1c75